src/frontend/components/Navbar/Navbar.jsx

```diff
@@ -16,6 +19,19 @@ const Navbar = () => {
         </div>
         <SearchBar />
         <div className="actions">
+          <button
+            className="btn-cta text-center"
+            onClick={() => {
+              if (isAuthenticated()) {
+                logout();
+                navigate("/signin");
+              } else {
+                navigate("/signin");
```

Comment:
instead of writing navigate two times here u can just write it once after ifelse